docs/main/user-guides/upgrades.md

```diff
@@ -21,18 +21,18 @@ Refer to the [Migration documentation](../user-guides/data-migration/migration-o
 
 ## Overview
 
-This upgrade flow allows the users to upgrade to the latest OpenEBS version 4.1.1 which is a unified installer for three Local Storages (a.k.a Local Engines):
+This upgrade process allows you to upgrade to the latest OpenEBS version 4.2 which is a unified installer for three Local Storages (a.k.a Local Engines):
```

Comment:
4.2 is not yet released

Reply:
Yes Vishnu! This documentation is update din the "Main" branch and not in the "4.1" branch. @avishnu